.ci/configure_crio_for_kata.sh

```diff
@@ -21,7 +21,7 @@ crio_config_file="/etc/crio/crio.conf"
 runc_flag="\/usr\/local\/bin\/crio-runc"
 kata_flag="\/usr\/local\/bin\/kata-runtime"
 
-minor_crio_version=$(crio --version | head -1 | cut -d '.' -f2)
+minor_crio_version=$(crio --version | egrep "[0-9]\.[0-9]+" | head -1 | sed 's/[^0-9.]*//g' | cut -d '.' -f2)
```

Comment:
since this is semver, I think you can use:  crio --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+"

Reply:
thanks for the suggestion, I changed it.